account_invoice_section_sale_order/models/sale_order.py

```diff
@@ -56,10 +58,14 @@ def _create_invoices(self, grouped=False, final=False, date=None):
                     )
                     sequence += 10
                 for move_line in self.env["account.move.line"].browse(move_line_ids):
```

Comment:
we overlooked here... this browse requires a sudo as well :face_holding_back_tears: